src/lib/y2network/sysconfig/interface_file.rb

```diff
@@ -65,6 +125,11 @@ def define_parameter(name, type = :string)
       #   @return [Integer] Length in bits for all keys used
       define_parameter(:wireless_key_length, :integer)
 
+      # @return [Integer] Number of supported keys
```

Comment:
how looks @return in constant case? in yardoc?

Reply:
it just displays the "return" type but, as it also shows the value, it is useless. Removed.